_posts/2019-05-30-thoughts-from-teaching-google-and-amazon-new-skills.md

```diff
@@ -9,36 +9,35 @@ categories:
+Getting started with Amazon was time consuming because of the enterprise and government. At that time, the partnership between the State of Utah and Amazon was very young and authentication and security systems needed to be put in place before we could begin our work. AWS and Alexa are two separate entities that cannot share credentials, so the State could not use the same account for Alexa and AWS. Restricting all of the IAM permissions for State developers to successfully deploy and debug while in an enterprise environment took some trial and error. While we waited for all of the pieces to fall into place, I used my personal Amazon account. All of the account permissions were straightforward and development progressed smoothly.
 
-Utah is a Google Drive Enterprise customer and has been for many years. Therefore, getting started with the Google Assistant was very simple. Every state employee has a Google account. All we needed was an "enterprise" project to start development so the billing was accurate. There were no delays or major hoops to jump through.
+Utah is a Google Drive Enterprise customer and has been for many years. Therefore, getting started with Google Assistant was very simple. Every State employee has a Google account. All we needed was an "enterprise" project to start development so the billing was accurate. There were no delays or major hoops to jump through.
 
 ## Development
 {: .text-left }
-
 ### Debug Cycle
 
-Having a quick debug test loop is paramount to developer productivity. When developing in the cloud, this is often hindered by the time taken for deployments and the lack of access to the developer tooling we are familiar with on our machines. I chose to use python and [Flask](http://flask.pocoo.org/) for the Alexa skill to keep a fast loop. Flask let us handle Alexa requests locally removing the time necessary for deploying to lambda. This was a _huge_ productivity boost as we were able to use the python debugger and all of the debugging tools we are familiar with. And it is pleasing to take a break from callbacks and promises once in a while.
+Having a quick debug test loop is paramount to developer productivity. When developing in the cloud, this is often hindered by the time taken for deployments and the lack of access to the developer tooling we are familiar with on our machines. I chose to use Python and [Flask](http://flask.pocoo.org/) for the Alexa skill to keep a fast loop. Flask let us handle Alexa requests locally, which reduced the time required for deploying to lambda. This was a _huge_ productivity boost as we were able to use the Python debugger and all of the debugging tools we are familiar with. And it is pleasing to take a break from callbacks and promises once in a while.
```

Comment:
it did not reduce the time. it eliminated the need to deploy to lambda and the time that takes.